exercises/10-exercise.py

```diff
@@ -90,14 +76,15 @@ def get_file_size(self, datum_gen):
 for i in range(len(datum_ids)):
     resource = db.reg.resource_given_datum_id(datum_ids[i])
     resources.append(resource)
-    #resources.add(resource['id'])
-   
+
 sizes = []
 # initialize a file handler
 # then retrieve the size for that specific file
 for resource in resources:
-	fh = NumpySeqHandler(resource['resource_path'], resource['root'])
-	datum_gen = db.reg.datum_gen_given_resource(resource)
-	sizes.append(fh.get_file_size(datum_gen))
-    
-print(sizes)
\ No newline at end of file
+    fh = NumpySeqHandler(resource['resource_path'], resource['root'])
+    datum_gen = db.reg.datum_gen_given_resource(resource)
```

Comment:
try:
```py
datum_kwargs_list = [datum['datum_kwargs'] for datum in datum_gen]
```

Reply:
It gives me an error, I'll push the updated code.
TypeError: format() argument after ** must be a mapping, not int

Reply:
in lines furhter above:
```index =kwargs['datum_kwargs']['index']```

change to:
```index =kwargs['index']```